test/utils_test.js

```diff
@@ -603,4 +627,23 @@ describe("utils", () => {
       expect(pad(4, 3)).eql("004");
     });
   });
+
+  describe("toErrorList()", () => {
+    it("should convert an errorSchema into a flat list", () => {
+      expect(toErrorList({
+        a: {
+          b: {
+            __errors: ["err1", "err2"]
+          }
+        },
+        c: {
+          __errors: ["err3"]
+        }
+      })).eql([
+        {stack: "b err1"},
+        {stack: "b err2"},
```

Comment:
In theory yes, but that would mean generating these paths. I may work on this, but I'm wondering about the added value. There's some for sure, but I'm pretty sure things would stay useable without full field path info. Need to think about it.